docs/framework/wcf/hosting-services.md

```diff
@@ -21,12 +21,12 @@ These hosting options range from running inside a console application to server
+ This hosting option consists of registering the application domain (AppDomain) that hosts a WCF service as a managed Windows Service (formerly known as NT service) so that the process lifetime of the service is controlled by the service control manager (SCM) for Windows services. Like the self-hosting option, this type of hosting environment requires that some hosting code is written as part of the application. The service is implemented as both a Windows Service and as a WCF service by causing it to inherit from the <xref:System.ServiceProcess.ServiceBase> class as well as from a WCF service contract interface. The <xref:System.ServiceModel.ServiceHost> is then created and opened within an overridden <xref:System.ServiceProcess.ServiceBase.OnStart%28System.String%5B%5D%29> method and closed within an overridden <xref:System.ServiceProcess.ServiceBase.OnStop> method. An installer class that inherits from <xref:System.Configuration.Install.Installer> must also be implemented to allow the program to be installed as a Windows Service by the Installutil.exe tool. For more information, see [How to: Host a WCF Service in a Managed Windows Service](./feature-details/how-to-host-a-wcf-service-in-a-managed-windows-service.md). The scenario enabled by the managed Windows Service hosting option is that of a long-running WCF service hosted outside of IIS in a secure environment that is not message-activated. The lifetime of the service is controlled instead by the operating system. This hosting option is available in all versions of Windows.
```

Comment:
```suggestion

This hosting option consists of registering the application domain (AppDomain) that hosts a WCF service as a managed Windows Service (formerly known as NT service) so that the process lifetime of the service is controlled by the service control manager (SCM) for Windows services. Like the self-hosting option, this type of hosting environment requires that some hosting code is written as part of the application. The service is implemented as both a Windows Service and as a WCF service by causing it to inherit from the <xref:System.ServiceProcess.ServiceBase> class as well as from a WCF service contract interface. The <xref:System.ServiceModel.ServiceHost> is then created and opened within an overridden <xref:System.ServiceProcess.ServiceBase.OnStart%28System.String%5B%5D%29> method and closed within an overridden <xref:System.ServiceProcess.ServiceBase.OnStop> method. An installer class that inherits from <xref:System.Configuration.Install.Installer> must also be implemented to allow the program to be installed as a Windows Service by the Installutil.exe tool. For more information, see [How to: Host a WCF Service in a Managed Windows Service](./feature-details/how-to-host-a-wcf-service-in-a-managed-windows-service.md). The scenario enabled by the managed Windows Service hosting option is that of a long-running WCF service hosted outside of IIS in a secure environment that is not message-activated. The lifetime of the service is controlled instead by the operating system. This hosting option is available in all versions of Windows.
```